core-bundle/tests/Doctrine/Backup/BackupTest.php

```diff
@@ -83,6 +84,6 @@ public function invalidFileNameProvider(): \Generator
 
     private function getValidBackupPath(): string
     {
-        return $this->getTempDir().'/valid_backup_filename__20211101141254.sql';
+        return Path::join($this->getTempDir(), '/valid_backup_filename__20211101141254.sql');
```

Comment:
Same issue here, the backup path will contain forward slashes, while the bath returned from the backup instance will contain backwards slashes on Windows. Using `Path::join` will automatically normalize the path.